app/src/org/commcare/views/dialogs/HorizontalPaneledChoiceDialog.java

```diff
@@ -15,6 +16,7 @@
 
     public HorizontalPaneledChoiceDialog(Context context, String title) {
         super(context, title);
+        dialog.requestWindowFeature(Window.FEATURE_NO_TITLE);
```

Comment:
titles are enabled by default with AppCompat, disabling them here for HorizontalPaneledChoiceDialog to keep them same as earlier.